packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-modal-content.spec.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import JurisdictionModalContent from '../jurisdiction-modal-content';
+import { findByDisplayValue, render, screen } from '@testing-library/react';
+import RootStore from 'Stores/index';
```

Comment:
can we sort this import statements as per this guide https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules

Reply:
@suisin-deriv line 2 should be the last import as it is relative path